src/WebJobs.Script.SiteExtension/WebJobs.Script.SiteExtension.proj

```diff
@@ -0,0 +1,171 @@
+    </ItemGroup>
+    <MSBuild Projects="@(_PublishPrivateProjectReferenceExistent)"
+      Targets="Publish"
+      BuildInParallel="$(BuildInParallel)"
+      Properties="NoBuild=true"
+      RemoveProperties="$(PublishPropertiesToRemove)"/>
+    <RemoveDir Directories="@(_PublishPrivateProjectReferenceExistent->'%(PublishPrivateDir)/workers/python')" />
+  </Target>
+
+  <Target Name="RemoveUnneededRuntimes">
+    <!-- These shouldn't exist since we build/publish with a windows runtime, but just in case. -->
+    <ItemGroup>
+      <_RuntimesToRemove Include="@(PublishRuntimeIdentifier->'$(SiteExtensionDir)%(PublishDir)/runtimes/linux')" />
+      <_RuntimesToRemove Include="@(PublishRuntimeIdentifier->'$(SiteExtensionDir)%(PublishDir)/runtimes/osx')" />
+      <_RuntimesToRemove Include="@(PublishRuntimeIdentifier->'$(PrivateSiteExtensionDir)%(PublishDir)/runtimes/linux')" Condition="%(PrivateExtension)" />
+      <_RuntimesToRemove Include="@(PublishRuntimeIdentifier->'$(PrivateSiteExtensionDir)%(PublishDir)/runtimes/osx')" Condition="%(PrivateExtension)" />
+    </ItemGroup>
+    <RemoveDir Directories="@(_RuntimesToRemove)" />
+  </Target>
+
+  <!-- Copies symbols from SiteExtension out so they can be distributed independently. -->
+  <Target Name="MoveSymbols">
+    <ItemGroup>
+      <_SymbolDirs Include="@(PublishRuntimeIdentifier->'$(SiteExtensionDir)%(PublishDir)')">
+        <Destination>$(PublishDir)Symbols/$(SiteExtensionName).Symbols.$(Version).%(Identity)</Destination>
+        <ZipArtifact>$(SiteExtensionName).Symbols.$(Version).%(Identity).zip</ZipArtifact>
+      </_SymbolDirs>
+      <_WorkerSymbols Include="$(SiteExtensionDir)workers/**/*.pdb" Destination="$(PublishDir)Symbols/$(SiteExtensionName).Symbols.$(Version).%(PublishRuntimeIdentifier.Identity)/workers" />
+      <ZipArtifact Include="@(_SymbolDirs->'%(Destination)')" TargetName="SiteExtensionSymbols/%(_SymbolDirs.ZipArtifact)" />
+    </ItemGroup>
+    <MoveSymbols Directories="@(_SymbolDirs)" Destinations="%(Destination)" />
+    <Copy SourceFiles="@(_WorkerSymbols)" DestinationFiles="%(Destination)/%(RecursiveDir)%(Filename)%(Extension)" SkipUnchangedFiles="true" />
+    <Delete Files="@(_WorkerSymbols)" />
+  </Target>
+
+  <!-- Remove all symbols (.pdb) from PrivateSiteExtension -->
+  <Target Name="DeletePrivateSymbols">
+    <ItemGroup>
+      <_PrivateSymbolsToRemove Include="$(PrivateSiteExtensionDir)/**/*.pdb" />
+    </ItemGroup>
+    <Delete Files="@(_PrivateSymbolsToRemove)" />
+  </Target>
+
+  <!-- Calculate all the file hashes for the SiteExtension -->
+  <Target Name="ComputeHardLinkHashes">
+    <ItemGroup>
+      <_FilesToHash Include="$(SiteExtensionDir)**" />
+    </ItemGroup>
+    <GetFileHash Files="@(_FilesToHash)" HashEncoding="base64">
+      <Output TaskParameter="Items" ItemName="_HashedFiles" />
+    </GetFileHash>
+  </Target>
```

Comment:
Are we skipping [this](https://github.com/Azure/azure-functions-host/blob/c6365fdbba0f895b24da785a9b2f31ad0a759530/build/build-extensions.ps1#L103)?